elections/code-of-conduct/2023/election.yaml

```
@@ -0,0 +1,18 @@
+name: 2023 Kubernetes Code of Conduct Committee Election
+organization: Kubernetes
+start_datetime: 2023-08-11 00:00:01
+end_datetime: 2023-08-17 11:59:59
+no_winners: 2
+allow_no_opinion: True
+delete_after: True
+election_officers:
+ - BenTheElder
+ - cblecker
+ - cpanato
+ - justaugustus
+ - mrbobbytables
+ - palnabarun
+ - tpepper
+eligibility: Eligibility is limited to the [current Steering Committee Members](https://github.com/kubernetes/steering/blob/c65bd4e586be59d12466b2bd6c19905240a97f0a/README.md#members)
+exception_description: No exception requests accepted for this election.
+exception_due: 2023-08-08 00:00:01
```

Hey, you've just uncovered a limitation in Elekto.

You can't omit `show_candidate_fields` entirely right now.  

Instead, you'll need to do something like:

```suggestion
exception_due: 2023-08-08 00:00:01
show_candidate_fields:
  - 
```

This will result in each candidate having a blank "None" field.  I've added fixing this as a feature request.